packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -41,6 +56,64 @@ export default class ReconnectionManager {
     this.configure(meeting);
   }
 
+  /**
+   * Sets the iceState to connected and clears any disconnect timeouts and
+   * related timeout data within the iceState.
+   *
+   * @returns {undefined}
+   * @public
+   * @memberof ReconnectionManager
+   */
+  iceReconnected() {
+    if (this.iceState.disconnected) {
+      LoggerProxy.logger.log('ReconnectionManager:index#iceReconnected --> ice has reconnected');
+
+      this.iceState.resolve();
+      this.iceState.resolve = () => {};
+
+      if (this.iceState.timer && this.iceState.timer.clearTimeout) {
+        this.iceState.timer.clearTimeout();
+      }
+
+      this.iceState.disconnected = false;
+    }
+  }
+
+  /**
+   * Set the iceState to disconnected and generates a timeout that waits for the
+   * iceState to reconnect and then resolves. If the ice state is already
+   * processing a reconnect, it immediately resolves. Rejects if the timeout
+   * duration is reached.
+   *
+   * @returns {Promise<undefined>}
+   * @public
+   * @memberof ReconnectionManager
+   */
+  waitForIceReconnect() {
+    if (!this.iceState.disconnected) {
+      LoggerProxy.logger.log('ReconnectionManager:index#waitForIceReconnect --> waiting for ice reconnect');
+
+      this.iceState.disconnected = true;
+
+      return new Promise((resolve, reject) => {
+        this.iceState.timer = setTimeout(() => {
```

Comment:
`setTimeout` returns a timer ID which is cleared by `clearTimeout(timerID)`. `timer.clearTimeout()` isn't a function in this case.